wire-ios/Wire-iOS Tests/ConversationMessageCell/ConversationCollapsedMessageCellTests.swift

```diff
@@ -0,0 +1,94 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import WireFoundation
+import XCTest
+@testable import Wire
+
+final class ConversationCollapsedMessageCellTests: ConversationMessageSnapshotTestCase {
```

Comment:
suggestion: by including "Snapshot" in the name, we know immediately that's a pure snapshot test suite.
```suggestion
final class ConversationCollapsedMessageCellSnapshotTests: ConversationMessageSnapshotTestCase {
```